pom.xml

```diff
@@ -41,7 +41,7 @@
     <hive.version>3.1.2</hive.version>
     <junit.jupiter.version>5.7.1</junit.jupiter.version>
     <license.maven.plugin.version>3.0</license.maven.plugin.version>
-    <tez.version>0.9.1</tez.version>
+    <tez.version>0.10.4</tez.version>
```

Comment:
We're using the version that Hive `3.1.2` depends on. I see Hive `3.1.3` has been released but it also depends on this older `tez` version. I don't mind moving to a newer version as long as all the tests etc. still pass. The alternative is for this to be overridden to the newer version in whatever project is using HiveRunner so this more closely matches what Hive provides by default, even if it's an older version.